lab_1_classify_by_unigrams/main.py

```diff
@@ -5,9 +5,22 @@
 """
 # pylint:disable=too-many-locals, unused-argument, unused-variable
 
+# откуда берется language(3) + округлить мсе + closest profile:/
+
 
 def tokenize(text: str) -> list[str] | None:
-    """
+    if isinstance(text, str):
+        text = text.lower
+        tokenized_list = []
+        for i in text():
```

Comment:
Amazing move, please try to find more usual way to call `lower`

Reply:
done